cypress/e2e/swap/errors.test.ts

```diff
@@ -116,4 +116,12 @@ describe('Swap errors', () => {
       getBalance(DAI).should('be.closeTo', initialBalance + 200, 1)
     })
   })
+
+  it('insufficient liquidity', () => {
+    cy.visit(`/swap?inputCurrency=${USDC_MAINNET.address}&outputCurrency=${DAI.address}`)
+    cy.get('#swap-currency-output .token-amount-input').type('100000000000000').should('have.value', '100000000000000') // 100 trillion
+    // The API takes a long and variable amount of time to return a 404, so setting the timeout to 60 seconds.
+    cy.contains('Insufficient liquidity for this trade.', { timeout: 60000 })
```

Comment:
this will potentially add a minute to our CI time - want to pair on getting `intercept` to work?